packit_service/worker/checker/bodhi.py

```diff
@@ -0,0 +1,42 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+
+from packit.config.aliases import get_branches
+
+from packit_service.constants import KojiBuildState
+
+from packit_service.worker.checker.abstract import Checker
+from packit_service.worker.handlers.mixin import GetKojiBuildEventMixin
+
+logger = logging.getLogger(__name__)
+
+
+class IsKojiBuildComplete(Checker, GetKojiBuildEventMixin):
+    def pre_check(self) -> bool:
+        """Check if builds are finished (=KojiBuildState.complete)
+        and branches are configured.
+        By default, we use `fedora-stable` alias.
+        (Rawhide updates are already created automatically.)
+        """
```

Comment:
e.g.here the checker class name doesn't correspond to the docstring and to what the pre_check checks, could we either rename it or split it to check the completeness and whether the branch we are reacting to is configured?

Reply:
You are right, they need to be splitted. There is also some more code, in the handlers, that can be removed using the Mixins. I have not done it yet, because I think that I have already touched too much code (the one needed to remove the celery argument from __init__) and it will be difficult to fix any bug introduced with this changes. I think that more refactoring should follow but we should do it gradually. So if you agree, for now, I just adjust the names and then I will create some more cards to finish all the refactor.